hawkbit-autoconfigure/src/main/java/org/eclipse/hawkbit/autoconfigure/security/SecurityManagedConfiguration.java

```diff
@@ -355,6 +357,7 @@ protected void configure(final AuthenticationManagerBuilder auth) throws Excepti
          *         of service protection filter in the filter chain
          */
         @Bean
+        @ConditionalOnProperty(prefix = "hawkbit.server.security.dos.filter", name = "enabled", matchIfMissing = true)
```

Comment:
There was no reason back in the day to configure them separately. UI simply has less requirements concerning amount of requests.